pkgs/applications/misc/calibre/default.nix

```diff
@@ -1,16 +1,20 @@
-{ stdenv, mkDerivation,  fetchurl, poppler_utils, pkgconfig, libpng
+{ lib, mkDerivation, fetchurl, poppler_utils, pkgconfig, libpng
 , imagemagick, libjpeg, fontconfig, podofo, qtbase, qmake, icu, sqlite
 , unrarSupport ? false, chmlib, python2Packages, libusb1, libmtp
-, xdg_utils, makeDesktopItem, wrapGAppsHook, removeReferencesTo, qt5
+, xdg_utils, makeDesktopItem, removeReferencesTo
 }:
 
+let
+  pypkgs = python2Packages;
```

Comment:
i would probably take `pythonPackages` as a parameter, and just set it explicitly in `all-packages.nix`. Would also make it a lot easier to override